compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/test/lowering_config_attr.mlir

```diff
@@ -49,15 +49,13 @@ module {
   func.func @test() attributes {
      compilation_info = #iree_codegen.compilation_info<
          lowering_config = <tile_sizes = []>,
-         translation_info = <CPUDefault>,
-         workgroup_size = [16, 4, 1],
-         subgroup_size = 32>} {
+         translation_info = <CPUDefault workgroup_size = [16, 4, 1] subgroup_size = 32>>} {
     return
   }
 }
 // CHECK: #config = #iree_codegen.lowering_config<tile_sizes = []>
-// CHECK: #translation = #iree_codegen.translation_info<CPUDefault>
-// CHECK: #compilation = #iree_codegen.compilation_info<lowering_config = #config, translation_info = #translation, workgroup_size = [16, 4, 1], subgroup_size = 32>
+// CHECK: #translation = #iree_codegen.translation_info<CPUDefault workgroup_size = [16, 4, 1] subgroup_size = 32>
```

Comment:
I'm surprised we don't have any commas between these

Reply:
adding the `,` makes the parser ambiguous. The parser is not a look ahead parser. So if I do the assembly format
```
(`,` `workgroup_size` `=` ...)? (`,` `subgroup_size` `=`)
```

on parsing the `,` it doesnt know whether it is parsing workgroup size or subgroup size. So I just dropped it